mllib/src/test/scala/org/apache/spark/ml/feature/UnivariateFeatureSelectorSuite.scala

```diff
@@ -0,0 +1,685 @@
+      (4.87955868, Vectors.dense(8.57631869e-01, 1.18279868e-01, 2.84428492e-01, 8.51544596e-01,
+        1.33220409e-02, 1.87044251e-01), Vectors.dense(8.57631869e-01, 1.18279868e-01)),
+      (1.45844834, Vectors.dense(2.43360773e-01, 4.83288948e-02, 1.10430569e-01, 4.33097852e-01,
+        5.63452248e-02, 8.24333214e-01), Vectors.dense(2.43360773e-01, 4.83288948e-02)),
+      (3.73352155, Vectors.dense(2.18226531e-01, 5.28477779e-01, 3.01852956e-01, 6.31664822e-04,
+        8.97463990e-01, 8.25297034e-01), Vectors.dense(2.18226531e-01, 5.28477779e-01)),
+      (7.15472802, Vectors.dense(6.95170305e-01, 7.35775299e-01, 4.32188618e-01, 2.26744166e-01,
+        5.13186095e-01, 2.91635657e-01), Vectors.dense(6.95170305e-01, 7.35775299e-01)))
+
+    val data_f_regression2 = Seq(
+      (4.38136774, Vectors.dense(0.21557113, 0.66070242, 0.89964323, 0.1569332, 0.84097522,
+        0.61614986), Vectors.dense(0.21557113)),
+      (2.7573449, Vectors.dense(0.14790391, 0.40356507, 0.2973803, 0.53051143, 0.35408457,
+        0.88180598), Vectors.dense(0.14790391)),
+      (4.10617119, Vectors.dense(0.39333276, 0.42790148, 0.41415147, 0.82478069, 0.57201431,
+        0.49972278), Vectors.dense(0.39333276)),
+      (4.61098326, Vectors.dense(0.46189165, 0.460305, 0.21054573, 0.16588781, 0.72898672,
+        0.41290627), Vectors.dense(0.46189165)),
+      (6.32149954, Vectors.dense(0.42527082, 0.83902909, 0.97275171, 0.76947383, 0.24470714,
+        0.57847281), Vectors.dense(0.42527082)),
+      (7.53246836, Vectors.dense(0.56185556, 0.94463811, 0.97741409, 0.27233834, 0.76460529,
+        0.53085766), Vectors.dense(0.56185556)),
+      (5.20573215, Vectors.dense(0.5828694, 0.45827703, 0.49305311, 0.13803643, 0.18242319,
+        0.14182515), Vectors.dense(0.5828694)),
+      (7.11513098, Vectors.dense(0.98848811, 0.43453809, 0.11712213, 0.4849829, 0.06431555,
+        0.76125387), Vectors.dense(0.98848811)),
+      (2.78159163, Vectors.dense(0.1181108, 0.43820753, 0.49576967, 0.75729578, 0.35355208,
+        0.48165022), Vectors.dense(0.1181108)),
+      (3.42803558, Vectors.dense(0.44250624, 0.24310088, 0.03976366, 0.24023351, 0.91659502,
+        0.75260252), Vectors.dense(0.44250624)))
+
+    val dataset_f_regression1 =
+      spark.createDataFrame(data_f_regression1).toDF("label", "features", "topFeature")
+
+    val dataset_f_regression2 =
+      spark.createDataFrame(data_f_regression2).toDF("label", "features", "topFeature")
+
+    val resultDF1 = FValueTest.test(dataset_f_regression1.toDF, "features", "label", true)
+    val resultDF2 = FValueTest.test(dataset_f_regression2.toDF, "features", "label", true)
+    val selector = new UnivariateFeatureSelector()
+      .setOutputCol("filtered")
+      .setFeatureType("continuous")
+      .setLabelType("continuous")
+    val indices1 = selector.selectIndicesFromPValues(6, resultDF1, "fdr", 0.1)
+    val indices2 = selector.selectIndicesFromPValues(6, resultDF2, "fdr", 0.1)
```

Comment:
Minor:
Why not directly construct a `resultDF`(only includes 2 column "pValue", "featureIndex") to test `selectIndicesFromPValues` ? This can simplify tests code.

Don't need to block merge.